cluster.tf

```
@@ -812,7 +813,7 @@ resource "google_container_node_pool" "pools" {
           local.node_pools_linux_node_configs_sysctls["all"],
           local.node_pools_linux_node_configs_sysctls[each.value["name"]]
         )
-        cgroup_mode = local.node_pools_cgroup_mode[each.value["name"]] == "" ? null : local.node_pools_cgroup_mode[each.value["name"]]
+        cgroup_mode = local.node_pools_cgroup_mode[each.value["name"]] == "" ? local.node_pools_cgroup_mode["all"] == "" ? null : local.node_pools_cgroup_mode["all"] : local.node_pools_cgroup_mode[each.value["name"]]
```

Thanks @DrFaust92! For clarity, I wonder if we could use `coalesce()`?

```suggestion
        cgroup_mode = coalesce(local.node_pools_cgroup_mode[each.value["name"], local.node_pools_cgroup_mode["all"], null)
```

---

Sure, I tried it before and it didnt work for some reason but now looks good. ill regen the modules